CONTRIBUTING.md

```diff
@@ -0,0 +1,156 @@
+# Essential Scala Collaborators
+
+Congratulations! You have been selected as a content owner for the  **Scala Essentials** nanodegree.
+
+
+## About the nanodegree
+
+The **Scala Essentials** nanodegree aims to provide a comprehensive tour of the Scala Programming Language for begginers. 
+
+You can see the design document [here](https://docs.google.com/document/d/1HZ1Rsn46x_BVAo8scSyVr9r5IesyTyHhGptjW1n_T6A/edit?usp=sharing).

```

Comment:
Should this document be public?

Reply:
Good catch @Rafailong ! No need for sharing this doc.